tests/Jobs/RockTheVote/ImportRockTheVoteRecordTest.php

```diff
@@ -3,12 +3,14 @@
 namespace Tests\Jobs\RockTheVote;
 
 use Tests\TestCase;
+use Chompy\SmsStatus;
 use Chompy\RockTheVoteRecord;
 use Chompy\Models\ImportFile;
 use Chompy\Models\RockTheVoteLog;
 use Chompy\Jobs\ImportRockTheVoteRecord;
 use DoSomething\Gateway\Resources\NorthstarUser;
 
+// @TODO: Replace all hardcoded RTV field names with RockTheVoteRecord constants to DRY.
```

Comment:
I generally prefer _not_ to have my tests rely on application code (like these constants), since it means an unintentional change in a constant might not fail a test. And this is probably more subjective, but I actually prefer to have my tests be less DRY – since it means you can read them more like independent "stories", rather than having to jump around to different files to piece together helpers and constants.